lisa/tools/lsblk.py

```diff
@@ -42,6 +43,7 @@ def __init__(
         used_blocks: int = 0,
         total_blocks: int = 0,
         percentage_blocks_used: int = 0,
+        fstype: str = "",
```

Comment:
why?

Reply:
self.fstype = fstype if fstype is not None else ""
any difference?

Reply:
I was think of something more like: fstype: Optional[str] =None. Though not sure if this could be possible here. Please feel free to close this if this is not possible